src/clojure/nrepl/middleware/print.clj

```diff
@@ -60,6 +61,9 @@
 (def configuration-keys
   [::print-fn ::stream? ::buffer-size ::quota ::keys])
 
+(def providable-keys
```

Comment:
I guess the difference between config-keys and providable keys will confuse many people.